config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -48,8 +48,7 @@ presubmits:
       testgrid-tab-name: pr-azure-sigs
   - name: json-sort-check
     decorate: true
-    always_run: false
-    optional: true
+    run_if_changed: 'images/capi/.*'
```

Comment:
I think this would be better as `images/capi/*\.json$` to only run when a JSON file is changed.

Reply:
Yup. Changed.